app/config/ViewConfig.scala

```diff
@@ -49,6 +53,10 @@ case class ViewConfig(
   val nonMtdUser = s"$frontendBaseUrl/vat-repayment-tracker/non-mtd-user"
   def vatVariationsUrl(vrn: Vrn) = s"${variationsUrlPrefix}/vat-variations/org/${vrn.value}/introduction"
 
+  def timeoutDialogSignOutUrl(implicit request: Request[_]) = if (RequestSupport.isLoggedIn)
+    controllers.routes.Controller.signout().url
```

Comment:
Does this need the `$frontendBaseUrl` so that it may work in non-local environments?

Reply:
`.url` builds a relative url which is similar to how other routing/redirections are done (when you stay within the service)

this Url doesn't get passed to other services as a callback param (where a relative url would fail if hosts differ)

Reply:
Brilliant, no worries then